.github/workflows/testLatestPublish.yml

```diff
@@ -0,0 +1,38 @@
+on:
```

Comment:
There are two ways this will run:
1. Every time a new tag is published
2. Every 5 hours

The sleep at the beginning of the action is to give the newly-published package some time to become available on npm. Otherwise `latest` may not target the most recent release.